src/App.js

```diff
@@ -11,7 +15,12 @@ class App extends React.Component {
   render() {
     return (
       <div className="App">
-        <Calculator />
+        <Navbar />
+        <Routes>
+          <Route exact path="/" element={<Home />} />
+          <Route path="/calculator" element={(<Calculator />)} />
```

Comment:
- [ optional] you can add more style and make the nav indicate which page you are currently on. nice work